README.md

```diff
@@ -1,17 +1,35 @@
-# Miranum-Website
+# Miranum Documentation 
 
-This repository contains the code our website is build with.
-The life version can be found under [https://miranum.com/](https://miranum.com/)
+This repository contains the Miranum documentation with can be found under [https://miranum.com/](https://miranum.com/)
 
-## development
+## Documentation Guidelines
+### PRs for every change
+All changes have to be done in a separate Branch. As soon as the changes are done please open a PR. A GitHub Action runs 
+with every commit to a Branch and checks if the documentation can be build. If you create a new branch make sure to name him according 
+to what you to (e.g. feat/xyz or fix/xyz). Please use semantic commit messages as described [here](https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716).
 
-You'd like to contribute? You're welcome!
+### Structure
+Name Markdown files according to the title. This makes it easier to find a file.
+Avoid non-alphanumeric characters in titles. Use the file name as internal document id to reference in the appropriate sidebars file.
```

Comment:
Use the filename as "an" internal document id to reference it in the appropriate sidebar file.